Directory.Packages.props

```diff
@@ -5,7 +5,8 @@
     <AspnetVersion>8.0.4</AspnetVersion>
     <MicrosoftExtensionsVersion>8.4.0</MicrosoftExtensionsVersion>
     <EfVersion>8.0.4</EfVersion>
-    <AspireVersion>8.0.0-preview.7.24251.11</AspireVersion>
+    <AspireVersion>8.0.0</AspireVersion>
+    <AspirePreviewVersion>8.0.0-preview.8.24258.2</AspirePreviewVersion>
```

Comment:
NIT: Probably a more accurate property name would be something like `AspireUnstablePackagesVersion` or something along those lines.